packages/dmn-js-drd/src/features/drill-down/DrillDown.js

```diff
@@ -93,6 +91,28 @@ export default class DrillDown {
     }
   }
 
+  _getOverlayHtml(className, title, enabled) {
+    const container = document.createElement('div');
+    container.className = 'drill-down-overlay';
+
+    if (!enabled) {
+      const icon = document.createElement('span');
+      icon.className = className;
+      container.appendChild(icon);
+
+      return container;
+    }
+
+    const button = document.createElement('button');
+    button.type = 'button';
+    button.className = className;
+    button.title = this._translate(title);
+
+    container.appendChild(button);
+
+    return container;
+  }
```

Comment:
What is the reason we are not using `domify` here? Personally, I find it easier to read than building with native API.

The function name can be confusing, as we return a DOM node and not HTML. We could consider renaming it to `_getOverlayNode`, but we do use it interchangeably in other places

Reply:
I wanted to reduce the risks of having accidental XSS vector. Note that `domify` doesn't care whether the result of `translate` call contains quotation marks which otherwise would have to be escaped.

Reply:
I renamed the method.

Reply:
Makes sense, let's keep it like this then 👍